app.py

```diff
@@ -11,7 +11,8 @@
 def index():
     item_list = []
     list_id = '5f56323626c33d81cd98b386'
-    r = Requests.get(f'https://api.trello.com/1/lists/{list_id}/cards?key={os.getenv("KEY")}&token={os.getenv("TOKEN")}')
+    url = f'https://api.trello.com/1/lists/{list_id}/cards?key={os.getenv("KEY")}&token={os.getenv("TOKEN")}'
```

Comment:
A hardcoded list id means your app will not work on other environments.

Reply:
but this is the list of the trello board, is it not meant to be linked to just one list?